packages/css/src/components/accordion/accordion.scss

```diff
@@ -3,14 +3,14 @@
  * Copyright Gemeente Amsterdam
  */
 
-@import "../../common/text-rendering";
+@use "../../common/text-rendering";
 
 .ams-accordion {
   display: flex;
   flex-direction: column;
   gap: var(--ams-accordion-gap);
 
-  @include text-rendering;
+  @include text-rendering.text-rendering;
```

Comment:
This looks weird. Should we do something about it?

I see `@use "…" as *;` is an option to remove the namespace. Duplicate names would throw an arror, but we can control that within our own library.

We could group related mixins into a single file with a conventional name to get e.g. `typography.text-rendering`.

Or we could accept this.